it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/AMLSControllerISpec.scala

```diff
@@ -214,17 +214,83 @@ class AMLSControllerISpec extends BaseISpec with SessionDataMissingSpec {
       await(sessionStoreService.fetchAMLSDetails) shouldBe empty
     }
 
-    "show validation error when the form is submitted with invalis expiry datge" in new Setup {
-      implicit val requst = authenticatedRequest.withFormUrlEncodedBody("amlsCode" -> "AAT",
+    "show validation error when the form is submitted with invalid expiry date" in new Setup {
+      implicit val request = authenticatedRequest.withFormUrlEncodedBody("amlsCode" -> "AAT",
         "membershipNumber" -> "12345", "expiry.day" -> "123", "expiry.month" -> expiryMonth,  "expiry.year" -> expiryYear)
 
-      val result = await(controller.submitMoneyLaunderingComplianceForm(requst))
+      val result = await(controller.submitMoneyLaunderingComplianceForm(request))
       status(result) shouldBe 200
       result should containMessages("moneyLaunderingCompliance.expiry.title", "error.moneyLaunderingCompliance.date.invalid")
 
       await(sessionStoreService.fetchAMLSDetails) shouldBe empty
     }
 
+    "show validation error when the form is submitted with empty day field" in new Setup {
+      implicit val request = authenticatedRequest.withFormUrlEncodedBody("amlsCode" -> "AAT",
+        "membershipNumber" -> "12345", "expiry.day" -> "", "expiry.month" -> expiryMonth,  "expiry.year" -> expiryYear)
+
+      val result = await(controller.submitMoneyLaunderingComplianceForm(request))
+      status(result) shouldBe 200
+      result should containMessages("moneyLaunderingCompliance.expiry.title", "error.moneyLaunderingCompliance.day.empty")
```

Comment:
Can we have a negative test assertion for other error messages here? eg: `result should not containMessages(...)`